client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -72,29 +67,26 @@ export const useInitialiseForm = () => {
     setIsLoading(true);
     setIsBootstrap(false);
     const syncSettings = {
-      centralServerSiteId: 1,
       intervalSec: 300,
       password,
-      siteId: siteId || 2,
       url,
       username,
     };
-
-    await update(syncSettings).catch(e => {
+    try {
```

Comment:
Not too sure about nested async call... but it does stop the server from restarting on sync error... 🤔